package.json

```diff
@@ -23,5 +21,11 @@
     "dev": "encore dev",
     "watch": "encore dev --watch",
     "build": "encore production --progress"
+  },
+  "dependencies": {
+    "add": "^2.0.6",
+    "bootstrap": "^4.3.1",
+    "tempusdominus-bootstrap-4": "^5.1.2",
```

Comment:
These should stay dev dependencies, as done before.

Reply:
corrected